services/models/route_rule.go

```diff
@@ -4,10 +4,33 @@ import (
 	"github.com/kiali/kiali/kubernetes"
 )
 
+// RouteRules routeRules
+//
+// This is used for returning an array of RouteRule
+//
+// swagger:model routeRules
+// An array of routeRule
+// swagger:allOf
 type RouteRules []RouteRule
+
+// RouteRule routeRule
+//
+// This is used for returning a RouteRule
```

Comment:
On these cases, where we are also describing practically an object from Istio, we can use a reference to their doc, too.
I mean, swagger should be able to enable these notes.
Developers can maintain them in an iterative way, but that is something we want to set up.